custom_components/tapo/tapo_sensor_entity.py

```diff
@@ -78,6 +78,26 @@ def native_value(self) -> StateType:
         return None
 
 
+class TapoMonthEnergySensor(TapoSensor):
+    def __init__(self, coordiantor: TapoUpdateCoordinator):
```

Comment:
Shouldn't it be coordinator instead of coordinator?

Reply:
???